opennms-webapp/src/main/webapp/index.jsp

```diff
@@ -63,7 +64,7 @@
 	<!-- Middle Column -->
 	<div class="col-md-6" id="index-contentmiddle">
 		<%
-			String centerUrl = System.getProperty("org.opennms.web.console.centerUrl", "/includes/categories-box.jsp");
+			String centerUrl = System.getProperty("org.opennms.web.console.centerUrl", "/includes/categories-box.jsp,/geomap/map-box.jsp");
```

Comment:
Please also change the default in the opennms.properties file: "The default is the view of SLM/RTC categories: /includes/categories-box.jsp."